entity-framework/core/get-started/install/index.md

```diff
@@ -123,7 +123,7 @@ For ASP.NET Core apps, this package is included automatically.
 
   * The .NET CLI tools are included in the .NET SDK since version 2.1, so the reference to that package can be removed from the project file:
 
-    ```
-    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" />
-    ```
+```xml
+<DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" />
+```
```

Comment:
Does this render the same? Indenting it made it part of the list item's body at one point

Reply:
Came outside list. I will make it inside.

Reply:
Went into the list 😉